network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/StaticVarCompensatorImpl.java

```diff
@@ -22,8 +22,11 @@
  */
 public class StaticVarCompensatorImpl extends AbstractInjectionImpl<StaticVarCompensator, StaticVarCompensatorAttributes> implements StaticVarCompensator {
 
+    private final RegulatingPoint regulatingPoint;
```

Comment:
Move up to AbstractIdentifiableImpl ? like for the regulation point attributes ...
And so for some other methods in this class ...

Reply:
yes I missed it sorry